src/openfermion/resource_estimates/pbc/__init__.py

```diff
@@ -0,0 +1,19 @@
+# coverage: ignore
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+import pytest
+
+try:
+    import pyscf
+except (ImportError, ModuleNotFoundError) as err:
+    pytest.skip(f"Need pyscf for PBC resource estimates {err}",
+                allow_module_level=True)
```

Comment:
what happens if you try to import this a) without `pyscf` installed and b) not in the context of running tests with pytest? genuinely curious.

Is pyscf installed in the CI?

Reply:
I did not know this but it throws an import/modulenotfound error (also the case for resource_estimates/__init__.py). This probably isn't very helpful. An alternative is to add pytest skipifs to the indivual unit tests conditional on the import, or I guess catch this throw from pytest.skip somehow and exit more gracefully. Seems like we need both.

pyscf is not, I think ideally we should conditionally run the tests for both this PBC code and the molecular resources (not run through CI), where conditionally means if this code changes. The reason is some tests here are quite slow and can't really be made much faster AND the molecular code has some external dependencies which may require a more complicated build instruction. I looked into it and there did seem to be some actions tools out there to only run conditional on code in certain paths changing, but wasn't sure if there was a more sensible way to do this. Anyway I will open an issue.

Reply:
punting this to #825